use-cases/predictive_maintenance/2_dataprep_predmaint.ipynb

```diff
@@ -4,114 +4,85 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you update this link text to be the doc title: <code>Common Data Formats for Training</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Done</p><p> </p><p> </p>